### PR TITLE
New metric for import duration (histogram).

### PR DESCRIPTION
**Pull request by ydayagi**

### CODE REVIEW

File: pkg/metrics/metrics.go [hunk 1]
```diff
@@ -24,49 +24,98 @@ var (
 		},
 		[]string{"result"},
 	)
-	// ImportCounter counter which hold number of vm imports done
-	ImportCounter = importCounter{importCounterVec}
+	// Histogram for duration of imports
+	importDurationVec = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name:    "kubevirt_vmimport_duration",
+			Help:    "Satistics of virtual machine import duration",
```
Comment (jakub-dzon) [inline pkg/metrics/metrics.go]: "Statistics"

**Review by jakub-dzon — COMMENTED** [review]
Overall looks good. 
Please fix the typo and make sure that the Travis build completes successfully.

File: pkg/metrics/metrics.go (hunk 1 above)
Comment (ydayagi) [reply pkg/metrics/metrics.go]: done

**Review by machacekondra — APPROVED** [review]

**Review by machacekondra — APPROVED** [review]

**Review by pkliczewski — APPROVED** [review]

**Review by jakub-dzon — APPROVED** [review]

**Pull request merged**